trino/sqlalchemy/dialect.py

```diff
@@ -363,11 +363,6 @@ def do_execute(
         self, cursor: Cursor, statement: str, parameters: Tuple[Any, ...], context: DefaultExecutionContext = None
     ):
         cursor.execute(statement, parameters)
-        if context and context.should_autocommit:
-            # SQL statement only submitted to Trino server when cursor.fetch*() is called.
-            # For DDL (CREATE/ALTER/DROP) and DML (INSERT/UPDATE/DELETE) statement, call cursor.description
-            # to force submit statement immediately.
-            cursor.description  # noqa
```

Comment:
can you move this to separate first commit since it's a cleanup that is not strictly related to SQLA 2.0 - it makes it easier to revert things or bisect in future.